docs/general/reporting_bugs.md

```diff
@@ -1,25 +1,27 @@
 ![SeAT](http://i.imgur.com/aPPOxSK.png)
 
+# Reporting Bugs
+
 So, the inevitable happened. Something broke! Its ok. The first thing to do is to relax, and prepare some info for a bug report.
 
-### the more info, the better
+### The more info, the better
 In order to best understand the bug, we need as much info as possible about your environment. For that, you can run the following command (from your SeAT directory), and copy / paste the output as part of your bug report:
 
 ```bash
 php artisan seat:admin:diagnose
```

Comment:
I would nudge users to use docker commands
`docker exec -it seat-app php artisan seat:admin:diagnose`

Reply:
i think the default should be docker this is all about nudging ppl :D

Reply:
Keep with a generic guide with docker as an optional install path for those who want it. Heading in a docker-only direction will drive people away, just like discourse.